securitas/messaging/schema.py

```diff
@@ -0,0 +1,34 @@
+from fedora_messaging import message
+
+class MemberSponsorV1(message.Message):
```

Comment:
I think this is right, just cant test it because of the entry points issue above